mg/cmd_internal_test.go

```diff
@@ -0,0 +1,75 @@
+package mg
+
+import (
+	"testing"
+)
+
+type nonSpec struct{ ActionType }
```

Comment:
In general, if you create a new action type and it's only used inside 1 function, it's best to just defined the type inside the function.

Reply:
Very valid point, thank you :smile: